oe_webtools.libraries.yml

```diff
@@ -0,0 +1,5 @@
+# Third-party library (CDN).
+drupal.webtools-smartloader:
```

Comment:
Shouldn't we prepend the library name with the name of the module ?

Reply:
@ademarco @sandervd What do you think about this topic?

Reply:
:man_shrugging: